kobo/apps/project_ownership/tests/api/v2/test_api.py

```diff
@@ -432,6 +432,7 @@ def test_account_usage_transferred_to_new_user(self):
         assert response.status_code == status.HTTP_201_CREATED
 
         # someuser should have no usage reported anymore
+        cache.clear()
```

Comment:
The thing is that we need to have cache activated during the first part of the test and after that we need to clear it, because this is after the project is transferred and we need the cache to stop reflecting the state previous to the transfer

Reply:
@Guitlle @noliveleger Should we consider the actual expected behavior here, rather than just the tests? If someone actually transfers a project, are they going get inaccurate usage data for however long the cache lasts? Maybe that's acceptable, but it might be good to discuss?